.aicoe-ci.yaml

```diff
@@ -1,12 +1,24 @@
 check:
-  - thoth-build
+  - overlays-build
+overlays_dir: "."
+overlays:
+  - name: builder-image
```

Comment:
As we are using the overlays method, and it is described in this PR
that the `overlays_dir` is `.`

then the pipeline would try to look for the sub_dir with name `builder-image` at the root
for the containerfile


and same fo the hitchhikers-guide.